test/policies/portfolio_policy_test.rb

```diff
@@ -4,12 +4,26 @@
 
 class PortfolioPolicyTest < ActiveSupport::TestCase
   def setup
-    classroom = create(:classroom)
-    @student = User.create!(username: "student", type: "Student", password: "password", classroom: classroom)
+    @classroom = create(:classroom)
+    @other_classroom = create(:classroom)
+    @student = User.create!(username: "student", type: "Student", password: "password", classroom: @classroom)
     @other_student = User.create!(username: "other_student", type: "Student", password: "password",
-                                  classroom: classroom)
+                                  classroom: @other_classroom)
+
+    @teacher = create(:teacher)       # Teacher who teaches @classroom
+    @other_teacher = create(:teacher) # Teacher who does NOT teach @classroom
+
+    @admin = create(:admin)
+
     @portfolio = Portfolio.create!(user: @student)
     @other_portfolio = Portfolio.create!(user: @other_student)
+
+    TeacherClassroom.create!(teacher: @teacher, classroom: @classroom)
+  end
+
+  test "admin can show any portfolio" do
+    assert PortfolioPolicy.new(@admin, @portfolio).show?
+    assert PortfolioPolicy.new(@admin, @portfolio_in_other_classroom).show?
```

Comment:
on line 19 we have `@other_portfolio` defined - is that what we should be using here? I don't see `@portfolio_in_other_classroom` defined anywhere.